src/commands/build.rs

```diff
@@ -10,11 +10,13 @@ use crate::site::Site;
 pub struct Build {
     #[arg(long, default_value = "./")]
     path: PathBuf,
```

Comment:
i wonder if these names will come back to bite us- maybe "project_root" and "config_path"?

Reply:
done!